api/modules/github/views.py

```diff
@@ -0,0 +1,43 @@
+import requests
+import requests_cache
+from datetime import timedelta
+
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from api.modules.github.github_response import ContributorResponse
+from api.modules.github.constants import GITHUB_API_GET_CONTRIBUTORS_URL
+
+hour_difference = timedelta(days=1)
+requests_cache.install_cache(expire_after=hour_difference)
+
+
+@api_view(['GET'])
+def get_contributors(request, project):
+    """
+    Return list of people contributed
+    :param request:
+    :param project:
+    :return: 503 if github api fails
+    :return: 200 successful
+    """
+    try:
+        api_response = requests.get(
+            GITHUB_API_GET_CONTRIBUTORS_URL.format(project)
```

Comment:
Use GitHub API codes as well
That increases the number of calls we can make
For testing you can use your own API key, for prod I will make and will deploy them